esbuild.mjs

```diff
@@ -33,6 +34,19 @@ let resolveMediaElement = {
 };
 
 async function main() {
+  // Generate translation types
+
+  const enJSONStr = fs.readFileSync("./src/locales/en-GB.json").toString();
+  const enJSON = JSON.parse(enJSONStr.replace(/"\$/g, '"'));
```

Comment:
Why the global replacement of dollar signs? This seems to have potential for side effects. If we don't want the dollar signs, should we just remove them from the files in the first place?